starter-code/views/gallery.hbs

```diff
@@ -0,0 +1,15 @@
+<section class="page">
+    <article class="gallery">
+        <div class="grid">
+            <div class="cell"><img src="/images/1473f5f8b4d2b444dfa4866d0f5b19ca.jpg" alt=""></div>
+            <div class="cell"><img src="/images/61CngygatqL._SL1500_.jpg" alt=""></div>
+            <div class="cell"><img src="/images/blooming-persian-lilac_53876-86166.jpg" alt=""></div>
+            <div class="cell"><img src="/images/maxresdefault.jpg" alt=""></div>
+            <div class="cell"><img src="/images/Michael-Scott-Misquotations-Office-Video.jpg" alt=""></div>
+            <div class="cell"><img src="/images/Products62075-1200x1200-1649647.jpg" alt=""></div>
+            <div class="cell"><img src="/images/SayNoToCilantro_400x400.png" alt=""></div>
+            <div class="cell"><img src="/images/this_is_fine.jpg" alt=""></div>
+            <div class="cell"><img src="/images/unnamed.jpg" alt=""></div>
```

Comment:
Here you could have put the images in an array and used the {{#each}} to display them : p